schema.sql

```diff
@@ -36,4 +36,26 @@ ALTER TABLE animals
 ADD CONSTRAINT fk_species FOREIGN KEY(species_id) REFERENCES species(id);
 
 ALTER TABLE animals
-ADD CONSTRAINT fk_owner FOREIGN KEY(owner_id) REFERENCES owners(id);
\ No newline at end of file
+ADD CONSTRAINT fk_owner FOREIGN KEY(owner_id) REFERENCES owners(id);
+
+-- Create a table named vets with the following columns
+CREATE TABLE vets (
+id integer PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+name varchar NOT NULL,
+age integer NOT NULL,
+date_of_graduation date NOT NULL);
+
+-- Create a "join table" called specializations to handle this relationship
+CREATE TABLE specializations (
+species_id INT NOT NULL,
+vets_id INT NOT NULL,
+CONSTRAINT fk_species FOREIGN KEY(species_id) REFERENCES species(id) ON DELETE CASCADE,
+CONSTRAINT fk_vets FOREIGN KEY(vets_id) REFERENCES vets(id) ON DELETE CASCADE
+);
+
+CREATE TABLE visits (
+animals_id INT NOT NULL,
+vets_id INT NOT NULL,
+CONSTRAINT fk_animals FOREIGN KEY(animals_id) REFERENCES animals(id) ON DELETE CASCADE,
```

Comment:
- [ ] Kindly, add the column `visit_date` with the data type `date` in the table `visits` as requested in the project requirements

Reply:
Yeah, thanks I forgot to add the line I used to create the 'visit_date' column